application/controllers/survey_response.py

```diff
@@ -39,11 +47,15 @@ def add_survey_response(self, case_id, file, file_name, survey_ref):
 
         if self.check_if_file_size_too_small(file_size):
             log.info('File size is too small')
-            return 'False'
-        elif not self.check_if_file_size_too_small(file_size):
-            log.info('File size is correct')
+            raise FileTooSmallError()
+        else:
             json_message = self._create_json_message_for_file(file_name, file_contents, case_id, survey_ref)
-            return send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
+            sent = send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
+            if not sent:
+                log.error("Unable to send file to rabbit queue")
+                raise SurveyResponseError()
+            else:
```

Comment:
This return else and return statement can be removed as you're no longer using in the surveys_response_view.py file.

I know it's used in the test_add_survey_response_success test but the the assertTrue can be removed there and replaced with:

try:
     self.survey_response.add_survey_response(case_id, file, filename, '023')
except Exception:
     self.fail("Should not raise an exception")